docs/cql/ddl.rst

```diff
@@ -289,6 +289,16 @@ For instance::
 
 The supported options are the same as :ref:`creating a keyspace <create-keyspace-statement>`.
 
+ALTER tablets-enabled KEYSPACE :label-caution:`Experimental`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Modifying a tablets-enabled keyspace is possible and there is no special CQL syntax introduced for that purpose, because in case a keyspace has tablets, altering it follows the same process as with vnode keyspaces, though there are some caveats:
+
+- the replication factor can be increased or decreased by at most 1 at a time - that being said, one can repeatedly modify RF to reach the desired value,
+- ``ALTER`` statement doesn't fully support `replication_factor` tag specified for `NetworkTopologyStrategy`, and it's safer to list DCs explicitly when altering a keyspace,
```

Comment:
we should make that official rather than declare it is risky and recommended not to be used.

Reply:
I'm not sure how to proceed, because in previous comments I am asked to expand replication_factor and if it's done, then this statement will be removed altogether. So either expand replication_factor in the source code and remove this sentence or correct this statement.

Reply:
Changed the docs to reject `replication_factor` tag when ALTERing a KS with Tablets.